stdlib/public/Concurrency/Task.swift

```diff
@@ -375,38 +392,26 @@ extension Task {
 
 }
 
-/// Run given throwing `operation` as part of a new top-level task.
-///
-/// Creating detached tasks should, generally, be avoided in favor of using
-/// `async` functions, `async let` declarations and `await` expressions - as
-/// those benefit from structured, bounded concurrency which is easier to reason
-/// about, as well as automatically inheriting the parent tasks priority,
-/// task-local storage, deadlines, as well as being cancelled automatically
-/// when their parent task is cancelled. Detached tasks do not get any of those
-/// benefits, and thus should only be used when an operation is impossible to
-/// be modelled with child tasks.
+/// Run the given non-throwing operation as part of a new top-level task.
 ///
-/// ### Cancellation
-/// A detached task always runs to completion unless it is explicitly cancelled.
-/// Specifically, dropping a detached tasks `Task.Handle` does _not_ automatically
-/// cancel given task.
+/// Avoid using a detached task unless it isn't possible
+/// to model the operation using structured concurrency features like child tasks.
+/// Child tasks inherit the parent task's priority,
+/// task-local storage, and deadlines,
```

Comment:
sadly we're not shipping deadlines this year

```suggestion
/// task-local storage, 
```

they'll come in the future though :)